tests/unit/backend/chalice/api_server/test_v1_submission.py

```diff
@@ -0,0 +1,62 @@
+import json
+import unittest
+
+from furl import furl
+
+from backend.corpora.common.corpora_orm import ProjectStatus
+from backend.corpora.common.entities import Project
+from tests.unit.backend.chalice.api_server import BaseAPITest
+from tests.unit.backend.utils import BogusProjectParams
+
+
+class TestSubmission(BaseAPITest, unittest.TestCase):
+    def test__list_submission__ok(self):
```

Comment:
Test for time intervals as well?

Reply:
the submission API does support to and from. Is that something we want?